perfkitbenchmarker/linux_benchmarks/hpcc_benchmark.py

```diff
@@ -192,6 +205,10 @@ def ParseOutput(hpcc_output, benchmark_spec):
   metadata['num_cpus'] = match.group(1)
   metadata['num_machines'] = len(benchmark_spec.vms)
   metadata['memory_size_mb'] = FLAGS.memory_size_mb
+  if FLAGS['hpcc_binary'].present:
+    metadata['override_binary'] = FLAGS.hpcc_binary
+  if FLAGS['hpcc_mpi_env'].present:
+    metadata['mpi_env'] = FLAGS.hpcc_mpi_env
```

Comment:
nitpick: I think there are enough lines here dealing with metadata to deserve its own helper function.